commons/src/main/kotlin/com/simplemobiletools/commons/dialogs/ExportBlockedNumbersDialog.kt

```diff
@@ -0,0 +1,81 @@
+package com.simplemobiletools.commons.dialogs
+
+import android.annotation.SuppressLint
+import androidx.appcompat.app.AlertDialog
+import com.simplemobiletools.commons.R
+import com.simplemobiletools.commons.activities.BaseSimpleActivity
+import com.simplemobiletools.commons.extensions.baseConfig
+import com.simplemobiletools.commons.extensions.beGone
+import com.simplemobiletools.commons.extensions.getCurrentFormattedDateTime
+import com.simplemobiletools.commons.extensions.getParentPath
+import com.simplemobiletools.commons.extensions.humanizePath
+import com.simplemobiletools.commons.extensions.internalStoragePath
+import com.simplemobiletools.commons.extensions.isAValidFilename
+import com.simplemobiletools.commons.extensions.setupDialogStuff
+import com.simplemobiletools.commons.extensions.toast
+import com.simplemobiletools.commons.extensions.value
+import com.simplemobiletools.commons.helpers.BLOCKED_NUMBERS_EXPORT_EXTENSION
+import com.simplemobiletools.commons.helpers.ensureBackgroundThread
+import java.io.File
+import kotlinx.android.synthetic.main.dialog_export_blocked_numbers.view.export_blocked_numbers_filename
+import kotlinx.android.synthetic.main.dialog_export_blocked_numbers.view.export_blocked_numbers_folder
+import kotlinx.android.synthetic.main.dialog_export_blocked_numbers.view.export_blocked_numbers_folder_label
+
+@SuppressLint("SetTextI18n")
+class ExportBlockedNumbersDialog(
+    val activity: BaseSimpleActivity, val path: String, val hidePath: Boolean,
```

Comment:
this is too confusing. Either have a new line for each argument, or put them all together. Also make sure you do Ctrl + Alt + O for optimizing imports in every file